### PR TITLE
Digital control chapter added

### PR DESCRIPTION
**Pull request by chetangk94**
Difference between Voltage Mode Control and Current Mode Control

### CODE REVIEW

**Comment by martinjaeger** [comment]
Please use commit messages and pull request titles that explain what has been added or changed instead of update_1 etc.

For commit messages, the first line should be a very brief summary. The following lines can be used for more detailed description what the commit includes.

File: docs/development/digital_control.md [hunk 1]
```diff
@@ -1 +1,127 @@
...
+        <img 
+            src = "./images/system_model.svg" 
+            alt = "system_model"
+            height = auto
+            width = "700px" />
+        <figcaption>Fig.3 Closed loop system</figcaption>
+    </center>
+</figure>  
+
+The closed-loop gain, from the PWM control effort (u) to the sensed output voltage (V<sub>sense</sub>) is given by
```
Comment (martinjaeger) [inline docs/development/digital_control.md:83]: I suggest to use in-line latex equation here ($V_{sense}$ to have the same font as in below equations.

File: .vscode/spellright.dict [hunk 2]
```diff
@@ -0,0 +1,3 @@
+analog_controller
```
Comment (martinjaeger) [inline .vscode/spellright.dict:1]: This file is not needed on the server and can be removed from the next commit.

**Review by martinjaeger — COMMENTED** [review]
Very nice text and images, only had a few small comments.

Will you add additional points to the discrete model chapter or should this be done later and I should merge the current status already?

**Comment by martinjaeger** [comment]
In order to understand the control better, probably also the DC/DC chapter is needed with at least the basic DC/DC schematic and some explanation. We can talk about this on Thursday.

File: .vscode/spellright.dict (hunk 2 above)
Comment (chetangk94) [reply .vscode/spellright.dict:1]: oh ya it is a mistake .... it is dictionary file

File: docs/development/digital_control.md (hunk 1 above)
Comment (chetangk94) [reply docs/development/digital_control.md:83]: ya sure

**Comment by martinjaeger** [comment]
Closing this PR, as there are two new separate branches for DC/DC converter and digital control.

**Pull request closed without merging**